build.sbt

```diff
@@ -1,10 +1,10 @@
-lazy val scala212 = "2.12.15"
+lazy val scala213 = "2.13.8"
 
-ThisBuild / scalaVersion := scala212
+ThisBuild / scalaVersion := scala213
 
-ThisBuild / version := "3.1.0"
+ThisBuild / version := "3.2.0"
```

Comment:
Could also go with 4.0.0 since it's not backwards compatible with prior version of scala 2.12.x